static/js/pages/independent-expenditures.js

```diff
@@ -66,6 +66,7 @@ $(document).ready(function() {
   new tables.DataTable($table, {
     title: 'Independent expenditure',
     path: 'schedules/schedule_e',
+    query: {is_notice: 'false'},
```

Comment:
Ah I see what's going on now. But I think we still don't need those added, as those are the aggregate tables not itemized tables, and we already exclude notices from those aggregates (correct?)